core/src/test/scala/com/salesforce/op/ModelInsightsTest.scala

```diff
@@ -474,4 +477,22 @@ class ModelInsightsTest extends FlatSpec with PassengerSparkFixtureTest {
     f0InDer3.variance shouldBe Some(3.3)
   }
 
+  it should "include raw feature distribution information calculated in RawFeatureFilter when it's included" in {
+    val wfRawFeatureDistributions = modelWithRFF.getRawFeatureDistributions()
+    val wfDistributionsGrouped = wfRawFeatureDistributions.groupBy(_.name)
+
+    // Currently, raw features that aren't explicitly blacklisted, but are not used because they are inputs to
```

Comment:
should this go to RawFeatureFilter docs?

Reply:
Sure I can move it there